File: src/main/java/com/lq/util/ResultUtil.java
package com.lq.util;


import com.alibaba.fastjson.JSON;
import com.lq.bean.Result;

public class ResultUtil {
    public static Object success(Object object){
        Result result=new Result(null);
        result.setCode(0);
        result.setMsg("成功");
        result.setData(object);
        return JSON.toJSON(result);
    }

    public static Object success(){
        return success(null);
    }

    public static Object error(){
        Result result=new Result(null);
        result.setCode(1);
        result.setMsg("失败");
        return JSON.toJSON(result);
    }

}

File: src/main/java/com/lq/util/ResultUtil.kt
package com.lq.util

import com.alibaba.fastjson.JSON
import com.lq.bean.Result
/**
 * Created by LiQian_Nice on 2018/3/9
 *
 * kotlin中的静态方法类
 */
object ResultUtil{

    /**
     * ResultUtil.INSTANCE.success(messageList)
     * Kotlin中为Any 等于 java中的Object
     */
    fun success(data:Any,code:Int,msg:String): Any? {
        var result = Result(data)
        result.msg = msg
        result.code = code
        return JSON.toJSON(result)
    }
}

